src/woo_publications/metadata/tests/test_theme_api_endpoints.py

```diff
@@ -113,18 +113,7 @@ def test_detail_theme(self):
     def test_detail_theme_wrong_uuid(self):
         list_url = reverse(
             "api:theme-detail",
-            kwargs={"uuid": str(uuid.uuid4)},
-        )
-
-        response = self.client.get(list_url, headers=AUDIT_HEADERS)
-
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
-
-    def test_detail_broken_uuid(self):
```

Comment:
I dropped this because we can't reverse URLs with invalid UUIDs :) the wrong UUID tests should cover broken UUIDs/404 behaviour